{{ cookiecutter.repo_name }}/src/main.py

```diff
@@ -0,0 +1,9 @@
+from fastapi import FastAPI
+
+app = FastAPI()
+
+
+@app.get("/")
+async def root():
+    """Return a simple message."""
+    return {"message": "Hello World"}
```

Comment:
This doesn't seem very useful, but with a little tweak it could be.

I would suggest changing this to be a `/health` endpoint that returns `{"status": "OK"}`, and include a message in the docstring to enhance the endpoint with application specific status information.